.github/labels.yml

```diff
@@ -86,3 +113,23 @@
 - name: "Difficulty: 🔴 hard"
   description: "Difficult task requiring significant resources and time."
   color: "DC3545"
+
+# OS
+- name: "OS: 💻 Independent"
+  description: "Case do not refer to any OS."
+- name: "OS: 🐧 Linux"
+  description: "Case refers to Linux."
+- name: "OS: 🪟 Windows"
+  description: "Case refers to Windows."
+
+# Engine type
+- name: "Engine: ⚙️ Independent"
+  description: "Case do not refer to any Engine."
+- name: "Engine: 📀 reHLDS"
+  description: "Case refers to reHLDS engine."
+- name: "Engine: 💿 HLDS"
+  description: "Case refers to original Valve's engine."
+- name: "Engine: 📺 reHLTV"
+  description: "Case refers to reHLDS version."
+- name: "Engine: 📼 HLTV"
+  description: "Case refers to original Valve's version."
```

Comment:
**Why is it necessary?**

If an issue is not Re* related - we will not mark it as relevant to the repository and the community can “just discuss” it in the discussion section.

Reply:
It important for statistics and some future automatisation. Why not?

Reply:
We could filter non relevant issues by this labels.

Reply:
They will be relevant as we use the issue **[transfer feature](https://docs.github.com/en/issues/tracking-your-work-with-issues/administering-issues/transferring-an-issue-to-another-repository#transferring-an-open-issue-to-another-repository)**between repositories within an organization.